### ISSUE TITLE
Comparing with paper performance

### ISSUE THREAD

**Issue by ninaqy** [orendv/learning_to_sample#5]
Hi, I used your training code "train_SNET.py" with "--num_out_points 32", after evaluating by "evaluate_SNET.py", it resulted in "eval accuracy: 0.47", which is much lower than your paper stated performance. In Fig 3 with sampling ratio=32, the paper states the accuracy should around 0.60. 

The same problem is found for progressiveNet, I failed to achieve the paper results by your providing running script.

I am wondering whether some special training tricks are involved, could you elaborate more on the training procedure? Thank you very much.

**Comment by orendv** [comment]
Hi, Did you follow the recipe in:
https://github.com/orendv/learning_to_sample/blob/master/classification/runner_SNET.sh
?
There is no trick, if you take this runner script and change 64 to 32 everywhere then you should indeed get an accuracy of ~60%

**Comment by itailang** [comment]
Dear ninachy,
Thank you for your interest in our project. This issue might be caused due to a problem in the training of the task network. I recommend to use the shell script 'runner_SNET.sh', or to follow the commands there, with the num_out_points that you want. Specifically, you should train the task network first, in this case, PointNet classifier. Use 'evaluate_classifier.py' to validate that the accuracy of the classifier is like that reported in PointNet paper (https://arxiv.org/abs/1612.00593).

**Comment by ninaqy** [comment]
Thanks for the suggestion. After training the classifier with 0.892 performance, now I can get similar results in your paper. 
Could you please also provide the calculation of NRE in the reconstruction task?

**Comment by itailang** [comment]
Yes. You should evaluate the Autoencoder first with complete input point clouds (using [this script](https://github.com/orendv/learning_to_sample/blob/master/reconstruction/autoencoder/evaluate_ae.py)). Then, run the evaluation script for S-NET (using [this script](https://github.com/orendv/learning_to_sample/blob/master/reconstruction/sampler/evaluate_s_net.py)) or ProgressiveNet (using [this script](https://github.com/orendv/learning_to_sample/blob/master/reconstruction/sampler/evaluate_progressive_net.py)). Note that you can also evaluate the Autoencoder with FPS sampled points. See detailed explanations at the [readme file for reconstruction](https://github.com/orendv/learning_to_sample/blob/master/reconstruction/README.md).
